packages/connect/src/api/blockchainEstimateFee.ts

```diff
@@ -44,6 +44,7 @@ export default class BlockchainEstimateFee extends AbstractMethod<'blockchainEst
                     { name: 'to', type: 'string' },
                     { name: 'txsize', type: 'number' },
                     { name: 'isCreatingAccount', type: 'boolean' },
+                    { name: 'creatingTokenAccountType', type: 'string' },
```

Comment:
nit: how about `newTokenAccountProgramName` so it'd be consistent with the rest of the naming in blockchain-link.